src/css/Partners.css

```
@@ -0,0 +1,127 @@
+   transition: background-position 0.5s, color 0.5s;
+}
+
+.partner-button:hover i::before,
+.partner-button:hover i::before {
+  content: "\F144";
+  font-family: 'bootstrap-icons';
+}
+
+.partner-button:hover {
+  background-position: 0 0;
+  background: linear-gradient(to bottom,#fbb147 , #f8713e);
```

Biome (1.9.4)

[error] Unexpected shorthand property background after background-position



(lint/suspicious/noShorthandPropertyOverrides)